README.md

```diff
@@ -106,6 +106,8 @@ Override some of the conventions:
 
 ```ruby
 Parity.configure do |config|
```

Comment:
I've never actually needed to override these conventions. My understanding was that you added an initializer to your application, but now that I'm looking at the code I'm not sure that's happening (and furthermore wouldn't play nicely with the Travelling Ruby packaging/isolation that was recently introduced). @croaky, where does the configuration get set?